packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -708,12 +763,343 @@ export const prepareVaultManagerKit = (
+          const accounting = liquidationResults(totalDebt, mintedProceeds);
+          const { Collateral: collateralProceeds } = proceeds;
+
+          const recordLiquidation = v => {
+            v.liquidated();
+            liquidatingVaults.delete(v);
+          };
+
+          const penaltyRate = facets.self
+            .getGovernedParams()
+            .getLiquidationPenalty();
+          // charged in collateral
+          const totalPenalty = ceilMultiplyBy(
+            totalDebt,
+            multiplyRatios(
+              penaltyRate,
+              quoteAsRatio(oraclePriceAtStart.quoteAmount.value[0]),
+            ),
+          );
+
+          // #7123 describes how to process liquidation proceeds
+          if (AmountMath.isEmpty(accounting.shortfall)) {
+            // Flow #1: no shortfall
+
+            const collateralToDistribute = AmountMath.isGTE(
+              collateralProceeds,
+              totalPenalty,
+            );
+
+            const distributableCollateral = collateralToDistribute
+              ? AmountMath.subtract(collateralProceeds, totalPenalty)
+              : AmountMath.makeEmptyFromAmount(collateralProceeds);
+
+            facets.helper.burnToCoverDebt(totalDebt, mintedProceeds, liqSeat);
+            facets.helper.sendToReserve(accounting.overage, liqSeat, 'Minted');
+
+            // return remaining funds to vaults before closing
+            const collateralPortion = makeRatioFromAmounts(
+              distributableCollateral,
+              totalCollateral,
+            );
+
+            /** @type {import('@agoric/zoe/src/contractSupport/atomicTransfer.js').TransferPart[]} */
+            const transfers = [];
+            let collatRemaining = distributableCollateral;
+
+            for (const [vault, amounts] of vaultData.entries()) {
+              facets.helper.subtractPhantomInterest(
+                amounts.debtAmount,
+                vault.getCurrentDebt(),
+              );
+              if (collateralToDistribute) {
+                const collat = floorMultiplyBy(
+                  amounts.collateralAmount,
+                  collateralPortion,
+                );
+                collatRemaining = AmountMath.subtract(collatRemaining, collat);
+                transfers.push([
+                  liqSeat,
+                  vault.getVaultSeat(),
+                  { Collateral: collat },
+                ]);
+              }
+              recordLiquidation(vault);
+            }
+            if (transfers.length > 0) {
+              atomicRearrange(zcf, harden(transfers));
+            }
+
+            facets.helper.liquidationUpdate(
+              AmountMath.subtract(totalCollateral, collateralProceeds),
+              vaultData.getSize(),
+            );
+            const sendToReserve = collateralToDistribute
+              ? AmountMath.add(collatRemaining, totalPenalty)
+              : collateralProceeds;
+            facets.helper.sendToReserve(sendToReserve, liqSeat);
```

Comment:
confusing to have a value with same name as the function it's passed to.
```suggestion
            facets.helper.sendToReserve(forReserve, liqSeat);
```

Reply:
fixed